examples/gnrc_border_router/Makefile.native.conf

```diff
@@ -1,16 +1,24 @@
+SHELL:=/bin/bash
+
+TAP = tapbr0
+
 # native has a TAP interface towards the host, just add parameters for
 # `socket_zep`
 ZEP_PORT_BASE ?= 17754
-ZEP_PORT_MAX := $(shell expr $(ZEP_PORT_BASE) + $(ZEP_DEVICES) - 1)
+ZEP_PORT_MAX := $(shell echo $$(($(ZEP_PORT_BASE) + 2 * $(ZEP_DEVICES) - 1)))
 
 CFLAGS += -DSOCKET_ZEP_MAX=$(ZEP_DEVICES)
 CFLAGS += -DASYNC_READ_NUMOF=$(shell expr $(ZEP_DEVICES) + 1)
 
 # Set CFLAGS if not being set via Kconfig
 CFLAGS += $(if $(CONFIG_KCONFIG_MODULE_DHCPV6),,-DCONFIG_DHCPV6_CLIENT_PFX_LEASE_MAX=$(ZEP_DEVICES))
 
-# -z [::1]:$PORT for each ZEP device
-TERMFLAGS += $(patsubst %,-z [::1]:%, $(shell seq $(ZEP_PORT_BASE) $(ZEP_PORT_MAX)))
+# -z [::]:$LOCAL_PORT,[::1]:$REMOTE_PORT for each ZEP device
+TERMFLAGS += $(shell paste -d ',' <(seq -f "-z [::1]:%g" $(ZEP_PORT_BASE) 2 $(ZEP_PORT_MAX))                     \
+                                  <(seq -f "[::1]:%g" $$(($(ZEP_PORT_BASE)+1)) 2 $$(($(ZEP_PORT_MAX)+1)))       \
+                                  | tr '\n' ' ')
```

Comment:
PR says this simplifies things, but can you please include an explainer here what is  happening here?

Reply:
Yea I noticed I need to use different local ports if there are multiple ZEP devices.

Unfortunately the documentation for ZEP is very sparse - is extending the local port to accept multiple connections the right way forward or am I missing something here?

Reply:
ZEP is UDP, there are no connections, so why should it need to be extended to accept multiple connection?

Reply:
> ZEP is UDP, there are no connections, so why should it need to be extended to accept multiple connection?

￼That's what I thought first too.
But it uses `real_connect` to bind to the remote socket and then only ever will read/write from that.

Starting a second client will only yield a "connection refused".
We would also need to keep a list of a clients  to send broadcast messages, this is not done.

So I think this was originally used in a different fashion than I anticipated.

Reply:
Mhh ah, because it was originally intended to be used with the server... It is indeed weird, that you _have_ to set the remote, but the local is optional.

https://github.com/RIOT-OS/RIOT/blob/f64511ddb5e6301beac5e16aae91f6fe34635f7c/cpu/native/startup.c#L298-L303

Even though I wrote that, in retrospect I find it weird.

Reply:
Ah wait. If you don't have a remote, the "radio" would have no way to know where to send the packet, so the `real_connect()` allows for that and you _need_ a server to allow for >2 nodes communicating that dispatches which nodes to send to.